magento/magento2
# Added Fix for - 26181

### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#26181: Out of stock text is not aligned properly with add to cart button at list page in responsive

### Manual testing scenarios (*)

1. Go to Frond end and product listing page
2. With this fix in mobile view the out of stock text and add links are aligned compared with add to cart button as shown below

![outofstok-expected](https://user-images.githubusercontent.com/54176640/71470552-8b3fbb00-27f2-11ea-8026-879214c4bcd1.png)


### Questions or comments


### Contribution checklist (*)
